src/UI/Status/Main.hs

```diff
@@ -65,11 +61,14 @@ renderStatusbar w s = withAttr statusbarAttr $ hBox
       )
   ]
 
-currentItemW :: L.List n e -> Widget n
-currentItemW l = str $
+currentItemW :: ListWithLength t e -> Widget n
+currentItemW (ListWithLength l len) = str $
   maybe
     "No items"
-    (\i -> "Item " <> show (i + 1) <> " of " <> total)
+    (\i -> "Item " <> show (i + 1) <> " of " <> maybe "?" show len)
```

Comment:
Hm... Perhaps in the instance of an unknown length show only the current index instead of a question mark? The question mark doesn't really provide any information but rather looks as if there is a bug.